terracotta/client/app/src/App.tsx

```diff
@@ -82,7 +82,7 @@ const App: FC<Props> = ({ hostnameProp }) => {
 
   const initializeApp = (hostname: string | undefined) => {
     // sanitize hostname
-    // hostname = 'https://4opg6b5hc3.execute-api.eu-central-1.amazonaws.com/development'
+    hostname = 'https://4opg6b5hc3.execute-api.eu-central-1.amazonaws.com/development'
```

Comment:
Is this only used during development? We should probably not include this in the code - or use an "official" demo deployment (which we currently are lacking).